src/chunkserver/datastore/file_pool.cpp

```diff
@@ -199,7 +204,9 @@ FilePool::FilePool(std::shared_ptr<LocalFileSystem> fsptr)
     : currentmaxfilenum_(0) {
     CHECK(fsptr != nullptr) << "fs ptr allocate failed!";
     fsptr_ = fsptr;
-    tmpChunkvec_.clear();
+    cleanAlived_ = false;
+    dirtyChunks_.clear();
```

Comment:
No need to clear it, it's empty by default.

Reply:
@wu-hanqing In some test case, it will call the `UnInitialize()` to do some clean work, and expect to clean these vector.